tests/test_bar_data.py

```diff
@@ -89,115 +88,100 @@ def check_internal_consistency(self, bar_data):
+            },
+            {
+                cls.HILARIOUSLY_ILLIQUID_ASSET_SID: create_minute_df_for_asset(
+                    cls.env,
+                    cls.bcolz_minute_bar_days[0],
+                    cls.bcolz_minute_bar_days[-1],
+                    50,
+                )
+            },
         )
 
     @classmethod
-    def tearDownClass(cls):
-        cls.tempdir.cleanup()
-
-    @classmethod
-    def create_adjustments_reader(cls):
-        path = create_mock_adjustments(
-            cls.tempdir,
-            cls.days,
-            splits=[{
+    def make_splits_data(cls):
+        return pd.DataFrame([
+            {
                 'effective_date': str_to_seconds("2016-01-06"),
                 'ratio': 0.5,
-                'sid': cls.SPLIT_ASSET.sid
-            }, {
+                'sid': cls.SPLIT_ASSET_SID,
+            },
+            {
                 'effective_date': str_to_seconds("2016-01-06"),
                 'ratio': 0.5,
-                'sid': cls.ILLIQUID_SPLIT_ASSET.sid
-            }]
-        )
-
-        return SQLiteAdjustmentReader(path)
+                'sid': cls.ILLIQUID_SPLIT_ASSET_SID,
+            },
+        ])
 
     @classmethod
-    def build_minute_data(cls):
-        market_opens = cls.env.open_and_closes.market_open.loc[cls.days]
-        market_closes = cls.env.open_and_closes.market_close.loc[cls.days]
+    def init_class_fixtures(cls):
+        super(TestMinuteBarData, cls).init_class_fixtures()
 
-        writer = BcolzMinuteBarWriter(
-            cls.days[0],
-            cls.tempdir.path,
-            market_opens,
-            market_closes,
-            US_EQUITIES_MINUTES_PER_DAY
+        cls.ASSET1 = cls.asset_finder.retrieve_asset(1)
+        cls.ASSET2 = cls.asset_finder.retrieve_asset(2)
+        cls.SPLIT_ASSET = cls.asset_finder.retrieve_asset(
+            cls.SPLIT_ASSET_SID,
         )
-
-        for sid in [cls.ASSET1.sid, cls.SPLIT_ASSET.sid]:
-            write_minute_data_for_asset(
-                cls.env,
-                writer,
-                cls.days[0],
-                cls.days[-1],
-                sid
-            )
-
-        for sid in [cls.ASSET2.sid, cls.ILLIQUID_SPLIT_ASSET.sid]:
-            write_minute_data_for_asset(
-                cls.env,
-                writer,
-                cls.days[0],
-                cls.days[-1],
-                sid,
-                10
-            )
-
-        write_minute_data_for_asset(
-            cls.env,
-            writer,
-            cls.days[0],
-            cls.days[-1],
-            cls.HILARIOUSLY_ILLIQUID_ASSET.sid,
-            50
+        cls.ILLIQUID_SPLIT_ASSET = cls.asset_finder.retrieve_asset(
+            cls.ILLIQUID_SPLIT_ASSET_SID,
+        )
+        cls.HILARIOUSLY_ILLIQUID_ASSET = cls.asset_finder.retrieve_asset(
+            cls.HILARIOUSLY_ILLIQUID_ASSET_SID,
         )
 
-        return BcolzMinuteBarReader(cls.tempdir.path)
+        cls.ASSETS = [cls.ASSET1, cls.ASSET2]
```

Comment:
It seems a bit odd that this is `cls.ASSETS` when we have two other named assets here.